### DISCUSSION TITLE
disable cache in Pimcore x

### DISCUSSION THREAD

**Discussion by langert** [pimcore/pimcore#9393] [Q&A / Support] [answered]
Hi,
how can the cache be disabled in pimcore x for developing? In the .env the APP_ENV=dev ist set to develop, but changes to templates need to refresh the opcache manually each time for showing up the results in editmode and frontend...

The output cache panel (shown in pimcore docs) does not exist in pimcore x any more?

![grafik](https://user-images.githubusercontent.com/11093410/121375449-0b9fa080-c941-11eb-82da-8eb01750ee5f.png)
Thx, Timo

**Comment by dvesh3** [comment, marked as answer]
@langert You have to define it in `config.yaml` like https://github.com/pimcore/demo/blob/10.x/config/config.yaml#L87-L91. I have also fixed the docs https://github.com/pimcore/pimcore/commit/25b1ff80883a5ce4aee1ae9a481e60cac44ac851 thanks!
